TaskCore.App/Options/ListTasksOptions.cs

```diff
@@ -7,11 +7,16 @@ public class ListTasksOptions: VerbOptionsBase
     {
         [OptionName("showcompleted", Alias = "c")]
         public bool ShowCompletedTasks { get; set; }
-        
         [OptionName("verbose",Alias = "v")]
         public bool Verbose { get; set; }
-        
         [OptionName("category", Alias="ca")]
         public string CategoryName { get; set; }
+        [OptionName("priority", Alias = "p")]
+        public int Priority { get; set; }
+
+        [OptionName("descendingduedate", Alias = "ddd", Description = "List by descending due date")]
+        public bool DuedateDescending { get; set; }
+        [OptionName("ascendingduedate", Alias = "asdd", Description = "List by ascending due date")]
+        public bool DuedateAscending { get; set; }
```

Comment:
Currently, the problem with this ordering is that if the user completes a task using the order number (which is presented as Task ID for simplicity), it will complete/close the wrong task because during task completion, to find out which actual task an order number matches, the code gets the active tasks in the default order `ls` is putting them out. You can check out the https://github.com/tarikguney/taskcore/blob/master/TaskCore.App/Verbs/CompleteTask.cs to see what exactly I am talking about.

**There are two solutions to this problem:**

1. Skip ordering by the due date until a better solution for handling completion is found. 
2. When ordered by the due date, the default order still determines the Order Number, but the order of tasks gets based on the due date selection. For instance:

```
3. [ ] Say hello to the World - 01/01/2020
1. [ ] Say hello to Mars - 01/02/2020
2. [ ]  Say hello to Jupiter - 01/03/2020
```

The numbers 3, 1, 2 are coming from the default ordering, but how the tasks are positioned in the output is determined by the due date flag.

If this is complicated or too much work, I would still appreciate the **filtering by priority** work, and if you delete the ordering code, I can merge the rest of the priority work in.

Thanks!

Reply:
Okay. I will delete this feature, re-commit and create a new branch to work about that.
Thanks for your time to description